shared/flexible-classroom/release-notes/android.mdx

```diff
@@ -1,5 +1,32 @@
 <PlatformWrapper platform="android">
 
+## v2.8.0
```

Comment:
Looks like all the release notes are the same. Please just use one file.

Reply:
They are not

Reply:
:-)